aws/resource_aws_glue_trigger.go

```diff
@@ -38,8 +38,14 @@ func resourceAwsGlueTrigger() *schema.Resource {
 							Optional: true,
 						},
 						"job_name": {
-							Type:     schema.TypeString,
-							Required: true,
+							Type:          schema.TypeString,
+							Optional:      true,
+							ConflictsWith: []string{"actions.0.crawler_name"},
```

Comment:
Noting here that while this happens to be that `actions` is a `TypeList` attribute and the `.0.` syntax will work with the first configuration block, it will not work with subsequent configuration blocks. Please see this upstream enhancement request: https://github.com/hashicorp/terraform-plugin-sdk/issues/71